2_gyum/aladin.py

```
@@ -0,0 +1,168 @@
+        try:
+
+            title = tree.select_one('.bo3')
+            title_text = title.text
+            title_link = title.attrs['href']
+
+            price = tree.select_one(".ss_p2").text
+            review = tree.select_one(".star_score").text
+            
+            datas.append([title_text, title_link, price, review])
+
+        except: continue
```

Ruff (0.13.1)

Do not use bare `except`

(E722)